README.md

```diff
@@ -287,51 +290,57 @@ Backoff strategy can be changed to `exponential` through constructor if necessar
 A connection to the host does not guarantee that your logs are transmitting
 successfully. If you have a bad token, there is no feedback from the server to
 indicate this. The only way to confirm that your token is working is to check
-the live tail on Logentries. I will investigate this further to see if there’s
+the live tail in the Insight Platform. I will investigate this further to see if there’s
```

Comment:
Does IDR have live tail?

Reply:
Nope doesn't seem like it, you prefer in `InsightOps` in this case?

Reply:
i guess it'd be better, more accurate anyway